docs/release/trg-0/trg-8-03.md

```diff
@@ -0,0 +1,257 @@
+---
+title: TRG 8.03 - KIT Artefacts
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 18-Jan-2024  | Initial contribution                   |
+
+This explains each artefacts a KIT can or must follow - depending on the graduation stage (TRG 8.02)
```

Comment:
artefacts again.

What does "artifacts, a KTI must follow" mean? Maybe "has to provide"?